.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -266,15 +266,46 @@ tsh --proxy=proxy.example.com --user=<username> --insecure ssh --cluster=foo.com
 
 - [ ] G Suite install instructions work
     - [ ] G Suite Screenshots are up to date
-- [ ] ActiveDirectory install instructions work
-    - [ ] Active Directory Screenshots are up to date
+- [ ] Azure Active Directory (AD) install instructions work
+    - [ ] Azure Active Directory (AD) Screenshots are up to date
+- [ ] ActiveDirectory (ADFS) install instructions work
+    - [ ] Active Directory (ADFS) Screenshots are up to date
 - [ ] Okta install instructions work
     - [ ] Okta Screenshots are up to date
 - [ ] OneLogin install instructions work
     - [ ] OneLogin Screenshots are up to date
+- [ ] GitLab install instructions work
+    - [ ] GitLab Screenshots are up to date
 - [ ] OIDC install instructions work
     - [ ] OIDC Screenshots are up to date
-
+- [ ] All providers with guides in docs are covered in this test plan
+
+### `tctl sso` family of commands
+
+`tctl sso configure` helps to construct a valid connector definition:
+
+- [ ] `tctl sso configure github ...` creates valid connector definitions
+- [ ] `tctl sso configure oidc ...` creates valid connector definitions
+- [ ] `tctl sso configure saml ...` creates valid connector definitions
+
+`tctl sso test` test a provided connector definition, which can be loaded from
+file or piped in with `tctl sso configure` or `tctl get --with-secrets`. Valid
+connectors are accepted, invalid are rejected with sensible error messages.
+
+- [ ] Connectors can be tested with `tctl sso test`.
+    - [ ] GitHub
+        - [ ] Teleport Cloud
+        - [ ] On-prem Teleport
```

Comment:
nit: The flows are totally the same so at fist glance testing with configurations both  envs setups  seems to be superfluous.

Reply:
Thinking about this further, I agree that this can be simplified. These are not Cloud-specific features in any way, so the requirement that these continue to work as usual falls under the general promise of Teleport Cloud to not break individual Teleport features. Done as of a730679a4c128d2ee1f0afcd9f78a1d0d942890a.